admob/integration_test/src/integration_test.cc

```diff
@@ -795,6 +795,8 @@ TEST_F(FirebaseAdMobTest, TestBannerView) {
   // Set the listener.
   TestBoundingBoxListener bounding_box_listener;
   banner->SetBoundingBoxListener(&bounding_box_listener);
+  PauseForVisualInspectionAndCallbacks();
```

Comment:
Wait to ensure that any callback attempts have time to complete before we start testing the listener.